docker-compose.yml

```diff
@@ -14,15 +20,50 @@ services:
       - ./.env:/app/.env
     networks:
       - app-network
-  postgresdb:
-    image: postgres:13.2
-    command: postgres -c 'max_connections=2000'
+  postgres_primary:
+    <<: *postgres-common
+    container_name: master
     ports:
       - "127.0.0.1:5432:5432"
     environment:
       POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: postgres
       POSTGRES_DB: main
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_HOST_AUTH_METHOD: "scram-sha-256\nhost replication all 0.0.0.0/0 md5"
+      POSTGRES_INITDB_ARGS: "--auth-host=scram-sha-256"
+    command: |
+      postgres 
+      -c wal_level=replica 
+      -c hot_standby=on 
+      -c max_wal_senders=10 
+      -c max_replication_slots=10 
+      -c hot_standby_feedback=on
+    volumes:
+      - ./init_replication.sql:/docker-entrypoint-initdb.d/init_replication.sql
```

Comment:
Hey, since init_replication.sql is small, do you think we can move this inside dockerfile or sh script? Harder to debug this way (you need to check one more file)

Reply:
+1 here

Reply:
I tried removing init_replication.sql file, but it doesn't work to run replication without it

Reply:
Yes, but you can try to embed it in .sh stage.. Like 

```
touch init_replication.sql
cat <sql> > init_replication.sql
```

Not very proficient with bash, but I think youll get the idea